pkgs/by-name/st/stalwart-mail/webadmin.nix

```diff
@@ -44,15 +44,19 @@ rustPlatform.buildRustPackage rec {
     nodejs
     npmHooks.npmConfigHook
     tailwindcss
-    trunk-ng
-    wasm-bindgen-cli
+    trunk
+    (wasm-bindgen-cli.override {
```

Comment:
We could possibly leave a small comment whats that about. e.G. that we need to match the version in upstreams cargo.lock and that it should be checked on a version bump.